lib/std/special/compiler_rt.zig

```diff
@@ -177,7 +179,9 @@ comptime {
         @export(@import("compiler_rt/arm.zig").__aeabi_memclr, .{ .name = "__aeabi_memclr4", .linkage = linkage });
         @export(@import("compiler_rt/arm.zig").__aeabi_memclr, .{ .name = "__aeabi_memclr8", .linkage = linkage });
 
-        @export(@import("compiler_rt/arm.zig").__aeabi_read_tp, .{ .name = "__aeabi_read_tp", .linkage = linkage });
+        if (builtin.os == .linux) {
```

Comment:
There's nothing linux-specific here, what error message did you get? I suppose it's because the GBA cpu doesn't support the `mrc` opcode we're using here.

Reply:
We only export that symbol under linux.

Reply:
But I can't see anything about that symbol being exclusive to Linux in the [specs](https://developer.arm.com/docs/ihi0043/e/run-time-abi-for-the-arm-architecture-abi-2018q4-documentation).